src/pykeen/nn/emb.py

```diff
@@ -237,3 +227,41 @@ def make(self, *, num_embeddings: int, device: Optional[torch.device] = None) ->
         if device is not None:
             rv = rv.to(device)
         return rv
+
+
+initializers = {
+    'xavier_uniform': xavier_normal_,
+    'xavier_normal': xavier_uniform_,
+    'xavier_normal_norm': compose(
+        nn.init.xavier_uniform_,
+        functional.normalize,
+    ),
+    'normal': torch.nn.init.normal_,
+    'uniform': torch.nn.init.uniform_,
+    'phases': init_phases,
+}
+
+constrainers = {
+    'normalize': functional.normalize,
+    'complex_normalize': complex_normalize,
+    'clamp': torch.clamp,
+    'clamp_normalize': clamp_norm,
+}
+
+# TODO add normalization functions
```

Comment:
Each normalizer can also be used as constrainer. There may be some constrainers which cannot be used as normalizer since they somehow break the autograd. However, as of now, all constrainers mentioned above could also be used as normalizers.

Reply:
We will come back to this once we have a reason to use a normalizer